model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -635,8 +637,18 @@ def model_output_to_completion_output(
                     tokens=tokens,
                 )
             except Exception as e:
-                logger.exception(f"Error parsing text-generation-inference output {model_output}")
-                raise e
+                logger.exception(f"Error parsing text-generation-inference output {model_output}. Error message: {json.loads(e)['error']}")
+                '''
+                if 'generated_text' not in model_output:
+                    raise ObjectHasInvalidValueException(
+                        f"Error parsing text-generation-inference output {model_output}. Error message: {json.loads(e)['error']}"
+                    )
+                '''
+                if model_output.get("error_type") == "validation":
+                    raise InvalidRequestException(model_output.get("error")) # trigger a 400
```

Comment:
i think some changes are needed in llms_v1.py, are you able to get 400 repro validation error with the current changes?

Reply:
yeah, added them so that we handle and return 400 HTTPException